mnist/README.md

```diff
@@ -1,170 +1,99 @@
-# Training MNIST using Kubeflow, S3, and Argo.
+# Training MNIST using Kubeflow, S3(Minio), and Argo.
```

Comment:
Why is S3 qualified using Minio? Does it not use S3 directly?

Reply:
in this example, I changed the s3 to minio since Minio is more easier for beginners and it does not depends on AWS account anymore.